data/net/minecraft/network/protocol/game/DebugPackets.mapping

```diff
@@ -35,6 +36,7 @@ CLASS net/minecraft/network/protocol/game/DebugPackets
 	METHOD sendPacketToAllPlayers (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/network/FriendlyByteBuf;Lnet/minecraft/resources/ResourceLocation;)V
 		ARG 0 level
 		ARG 1 buffer
+		ARG 2 location
```

Comment:
Better to follow `ClientboundCustomPayloadPacket`'s name for this, which differentiates it from positional location.
```suggestion
		ARG 2 identifier
```